3-Register/src/app/signed-in/[username]/loading.tsx

```diff
@@ -0,0 +1,9 @@
+import { Loading } from "@/components/Loading";
+
+export default function Loader() {
+  return (
+    <main className="flex flex-col flex-1 items-center justify-center p-16 gap-4">
+      <Loading />
+    </main>
+  );
+}
```

Comment:
I think it's a little weird having a `main` in the `Loader`, since one would think it's a very low-level component. Maybe we can place the `main` in the layout, so we make sure there is always just one?